js/gateio.js

```diff
@@ -656,15 +662,24 @@ module.exports = class gateio extends Exchange {
                         // Fee is in %, so divide by 100
                         'taker': this.parseNumber (Precise.stringDiv (takerPercent, '100')),
                         'maker': this.parseNumber (Precise.stringDiv (makerPercent, '100')),
-                        'contractSize': this.safeString (market, 'contractSize', '1'),
+                        'contractSize': this.safeString (market, 'quanto_multiplier'),
+                        'precision': {
+                            'amount': 1,
+                            'price': undefined,
```

Comment:
The price precision is required (line 2061).

Reply:
how do we parse 0.1 price precision?

Reply:
We set precisionMode = TICK_SIZE and use increments instead of decimal integers for precision everywhere throughout all methods using amountToPrecision, priceToPrecision, currencyToPrecision, etc. Cmd+f `'precisionMode': TICK_SIZE` around the repo.

Reply:
We might eventually switch to TICK_SIZE everywhere (except a few rare cases where SIGNIFICANT_DIGITS is used).

Reply:
If price precision is always a power of ten, we can use the default DECIMAL_PLACES precisionMode.